src/main/java/com/j256/ormlite/stmt/query/ManyClause.java

```diff
@@ -51,22 +51,30 @@ public ManyClause(Clause[] others, Operation operation) {
 
 	@Override
 	public void appendSql(DatabaseType databaseType, String tableName, StringBuilder sb,
-			List<ArgumentHolder> selectArgList) throws SQLException {
-		sb.append('(');
-		first.appendSql(databaseType, tableName, sb, selectArgList);
+			List<ArgumentHolder> selectArgList, Clause outer) throws SQLException {
+		boolean closing;
+		if (outer instanceof ManyClause && ((ManyClause) outer).operation == operation) {
+			closing = false;
+		} else {
+			sb.append('(');
+			closing = true;
+		}
```

Comment:
Consider rewriting:

```java
boolean closing = !(outer instanceof ManyClause && ((ManyClause) outer).operation == operation);
if (closing) {
  sb.append('(');
}
```

Reply:
Ha!  I flip flopped over that particular line but I like the look of yours.  Hummm.